File: solution/ServerExam/Entity/PaperEntity.cs
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace ServerExam.Entity
{
    public class PaperEntity
    {
        [Key]
        [JsonPropertyName("ID")]
        public int ID { get; set; }

        /// <summary>
        /// 试卷名称
        /// </summary>
        [JsonPropertyName("PaperName")]
        public string PaperName { get; set; }

        /// <summary>
        /// 科目ID
        /// </summary>
        [JsonPropertyName("SubjectID")]
        public int SubjectID { get; set; }

        /// <summary>
        /// 总分
        /// </summary>
        [JsonPropertyName("TotalScore")]
        public decimal TotalScore { get; set; }

        /// <summary>
        /// 及格线
        /// </summary>
        [JsonPropertyName("PassLine")]
        public decimal PassLine { get; set; }

        /// <summary>
        /// 考试时长
        /// </summary>
        [JsonPropertyName("ExamDuration")]
        public int ExamDuration { get; set; }

        /// <summary>
        /// 试卷状态 1正常 2禁用
        /// </summary>
        [JsonPropertyName("PaperState")]
        public int PaperState { get; set; }

        /// <summary>
        /// 创建时间
        /// </summary>
        [JsonPropertyName("CreateTime")]
        public int CreateTime { get; set; }

        /// <summary>
        /// 更新时间
        /// </summary>
        [JsonPropertyName("UpdateTime")]
        public int UpdateTime { get; set; }

        public PaperEntity()
        {
            this.ID = 0;
            this.PaperName = "";
            this.SubjectID = 0;
            this.TotalScore = 0;
            this.PassLine = 0;
            this.ExamDuration = 0;
            this.PaperState = 0;
            this.CreateTime = 0;
            this.UpdateTime = 0;
        }
    }
}

File: solution/Entity/PaperEntity.py
from Entity.BaseEntity import *


# 试卷
class PaperEntity(BaseEntity, BaseORM):
    __tablename__ = 'Paper'

    PaperName: str = Column(String(128), comment='试卷名称', default='none')
    PaperCode: str = Column(String(128), index=True, comment='试卷编码', default='none')
    SubjectID: int = Column(INTEGER, comment='科目ID', default=0)
    TotalScore: float = Column(DECIMAL(10, 2), comment='总分', default=0.00)
    PassLine: float = Column(DECIMAL(10, 2), comment='及格分数', default=0.00)
    ExamDuration: int = Column(INTEGER, comment='考试时长', default=0)
    PaperState: int = Column(INTEGER, comment='试卷状态 1正常 2禁用', default=0)
    UpdateTime: int = Column(INTEGER, comment='更新时间', default=int(time()))

    def __init__(self):
        super().__init__()